xbmc/Application.cpp

```diff
@@ -4301,7 +4301,9 @@ void CApplication::ProcessSlow()
 
   g_mediaManager.ProcessEvents();
 
-  CInputManager::Get().EnableRemoteControl();
+  if (CInputManager::Get().IsRemoteControlEnabled() &&
+      !CInputManager::Get().IsRemoteControlInitialized())
+    CInputManager::Get().EnableRemoteControl();
```

Comment:
Either this is wrong or there is something wrong with the naming.
```
if (enabled && ...)
  enable
```
just doesn't sound right to me. Maybe it should be `InitializeRemoteControl()` instead?